x-pack/packages/kbn-elastic-assistant-common/docs/openapi/serverless/elastic_assistant_api_2023_10_31.bundled.schema.yaml

```diff
@@ -0,0 +1,114 @@
+openapi: 3.0.3
+info:
+  description: Manage and interact with Security Assistant resources.
+  title: Security AI Assistant API (Elastic Cloud Serverless)
+  version: '2023-10-31'
+servers:
+  - url: 'http://{kibana_host}:{port}'
+    variables:
+      kibana_host:
+        default: localhost
+      port:
+        default: '5601'
+paths:
+  /api/elastic_assistant/chat/complete:
```

Comment:
```suggestion
  /api/security_ai_assistant/chat/complete:
```

Reply:
This is generated from the other specs, so will update automatically.